config.toml

```diff
@@ -12,10 +12,10 @@ aim = 'We aim to assist struggling Malaysian youth who lack academic support pri
 tutorSignUpLink = 'https://docs.google.com/forms/d/e/1FAIpQLSf6AkKHe-03225EL0eMiKribR0APFzCh8aykc1IB509oC9LZg/viewform?usp=sharing'
 tuteeSignUpLink = 'https://docs.google.com/forms/d/e/1FAIpQLSfpkixo2t9NKedNYlEChe2UvtnuVN9hTkMM6NcrkPNSIPv5SQ/viewform?usp=sharing'
 pages = [
-    ['Home', ''],
     ['About', 'about'],
     ['Projects', 'projects'],
-    ['Resources', 'resources']
+    ['Resources', 'resources'],
+    ['Guideline', 'guideline']
```

Comment:
About the navbar, I think it would be better to merge all the PRs first. If I change it here in my PR, it will cause merge conflicts with other PRs that don't have the latest version of the navbar. So, let's merge all the PRs first, then open a new issue to fix the navbar's responsiveness.